test-cases/longevity/longevity-100gb-4h.yaml

```diff
@@ -26,3 +26,6 @@ client_encrypt: true
 
 pre_create_schema: True
 sstable_size: 100
+
+hinted_handoff: 'enabled'
```

Comment:
Maybe better to reuse the `test-cases/features/hinted-handoff.yaml` file in the appropriate jenkins file adding to this one?
In this case it will be more obvious what we run - we will see in the jenkins pipeline parameters.
Also, we will be able to remove it's usage from a test run not changes the source code.

WDYT?

Reply:
On one hand, you're right that it's more controllable and it will be done only in the pipelines intended to.
OTOH, it's much harder to know that this change is there and to start adding it to all 4h pipelines.

Reply:
I prefer it like this for now.